src/components/Instructions/useInstructionsMessage.test.js

```diff
@@ -18,6 +18,18 @@ jest.mock('utils', () => ({
   isXblockStep: jest.fn(),
 }));
 
+jest.mock('@edx/frontend-platform/i18n', () => ({
+  ...jest.requireActual('@edx/frontend-platform/i18n'),
+  useIntl: () => ({
+    formatMessage: (message, values) => {
+      if (message.defaultMessage && values) {
+        return message.defaultMessage.replace('{earned}', values.earned).replace('{possible}', values.possible);
+      }
+      return message.defaultMessage;
+    },
+  }),
+}));
```

Comment:
Does using the `formatMessage` from `testUtils.jsx` not work here? It seems like the logic should be almost the same (the only thing that stands out as different is this one having the check for `message.defaultMessage` being falsy)